airflow/operators/bash_operator.py

```diff
@@ -78,7 +80,7 @@ def execute(self, context):
                     raise AirflowException("Bash command failed")
 
         if self.xcom_push:
-            return str(line.strip())
+            return str(line.strip(), sys.getdefaultencoding())
```

Comment:
Would `"{}".format(line.strip())` work here? I'd like to use format consistently across the board since it seems to be the most elegant yet reliable way to cast to a string across 2.7 / 3.4+